client/web/antrea-ui/src/utils/fakeData.ts

```diff
@@ -0,0 +1,94 @@
+import { AgentInfo } from '../api/info';
+
+export function generateFakeAgents(n: number): AgentInfo[] {
+    // Helper function to generate random agent names
+    const generateAgentName = () => {
+        const prefixes = ['antrea', 'k8s', 'node', 'worker', 'master'];
+        const suffixes = ['-agent', '-node', '-vm', '-host', ''];
+        const prefix = prefixes[Math.floor(Math.random() * prefixes.length)];
+        const suffix = suffixes[Math.floor(Math.random() * suffixes.length)];
+        const id = Math.random() < 0.3 ? 
+            Math.random().toString(36).substring(2, 6) : // alphanumeric
+            Math.floor(Math.random() * 1000).toString(); // numeric
+        return `${prefix}${suffix}-${id}`;
+    };
+
+    // Helper function to generate random versions
+    const generateVersion = () => {
+        const major = Math.floor(Math.random() * 3);
+        const minor = Math.floor(Math.random() * 10);
+        const patch = Math.floor(Math.random() * 20);
+        return `v${major}.${minor}.${patch}`;
```

Comment:
here again, I see little value in having totally random version numbers
I would actually suggest having 2 possible different version numbers (e.g., `2.2.3` and `2.3.0`), which would be representative of what you could have when you are in the middle of updating Antrea in a large cluster